src/tests/ftest/util/systemctl_utils.py

```diff
@@ -0,0 +1,247 @@
+"""
+  (C) Copyright 2018-2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import getpass
+import os
+import tempfile
+
+from ClusterShell.NodeSet import NodeSet
+# pylint: disable=import-error,no-name-in-module
+from util.file_utils import create_directory, distribute_files
+from util.run_utils import command_as_user, run_remote
+
+
+class SystemctlFailure(Exception):
+    """Base exception for this module."""
+
+
+def get_service_status(logger, hosts, service):
+    """Get the status of the daos_server.service.
+
+    Args:
+        logger (Logger): logger for the messages produced by this method
+        hosts (NodeSet): hosts on which to get the service state
+        service (str): name of the service
+
+    Returns:
+        dict: a dictionary with the following keys:
+            - "status":       boolean set to True if status was obtained; False otherwise
+            - "stop":         NodeSet where to stop the daos_server.service
+            - "disable":      NodeSet where to disable the daos_server.service
+            - "reset-failed": NodeSet where to reset the daos_server.service
+
+    """
+    status = {
+        "status": True,
+        "stop": NodeSet(),
+        "disable": NodeSet(),
+        "reset-failed": NodeSet()}
+    status_states = {
+        "stop": ["active", "activating", "deactivating"],
+        "disable": ["active", "activating", "deactivating"],
+        "reset-failed": ["failed"]}
+    command = ["systemctl", "is-active", service]
```

Comment:
This will need similar --user logic like get_systemctl_command

Reply:
Fixed